RELEASE/scripts/autoscend.ash

```diff
@@ -1,4 +1,4 @@
-since r20749;	//min mafia revision needed to run this script. Last update: Fix Quantum Terrarium Request parsing breaking on melodramedary
+since r20762;	//min mafia revision needed to run this script. Last update: Add 'logPreferenceChangeFilter', which is a comma separated list of preferences to not log when logPreferenceChange is enabled
```

Comment:
Since this is just a preference I don't think we really need to update the required version. Worst case it will continue running and be back to spamming `auto_maximize_current` changes. But we can go with it regardless if you just want to force it for QoL.

Reply:
I'll leave it. There's maximizer and familiar scrapbook fixes between those two we'll force people to take advantage of if we make them update.